src/main/resources/static/javascript/mainApp/grouping.controller.js

```diff
@@ -624,7 +694,7 @@
         $scope.existsInList = (listName, members) => {
             const currentPage = getCurrentPage(listName);
             const membersInList = members.filter((member) =>
-                _.some(currentPage, {uhUuid: member}) || _.some(currentPage, {uid: member})
+                _.some(currentPage, {uhUuid: member}) || _.some(currentPage, {uid: member}) || _.some(currentPage, {name: $scope.groupingName})
```

Comment:
Just a question, what does the `|| _.some(currentPage, {name: $scope.groupingName}` do?